r6p2/src/devicedrv/mali/common/mali_mem_validation.c

```diff
@@ -59,7 +59,9 @@ _mali_osk_errcode_t mali_mem_validation_check(u32 phys_addr, u32 size)
 		}
 	}
 
-	MALI_PRINT_ERROR(("MALI PHYSICAL RANGE VALIDATION ERROR: The range supplied was: phys_base=0x%08X, size=0x%08X\n", phys_addr, size));
+	MALI_PRINT_ERROR(("MALI PHYSICAL RANGE VALIDATION ERROR: The range supplied was:\n"
+			    "phys_base=0x%08X, size=0x%08X, valid_base=0x%08X, valid_size=0x%08X\n",
+			    phys_addr, size, mali_mem_validator.phys_base, mali_mem_validator.size));
```

Comment:
You shouldn't modify the file directly.

And the error message still isn't really clear. Is the range supplied the allowed range supplied through the DT, or the one provided to the driver by the blob?

Reply:
Oh, you're right. It has to be a separate patch.
About error I assumed it is range provided through DT,
but I'm not sure, I have to test it.

So I prefer to remove this commit and postpone it.